### DISCUSSION TITLE
Reexecute a failed run

### DISCUSSION THREAD

**Discussion by TimoE93** [dagster-io/dagster#4784] [Q&A] [answered]
Hi Guys,

Is it possible to reexecute a failed run by runid?

I tried the reexecute_pipeline function, which needs the parent run ID. But the most of my failed runs do not have a parent run ID, Just the runid.

I couldn' t find any function or graphql mutation to reexecute the failed run.

Do you have any Idea?

Thank you very much in advance

**Comment by alangenfeld** [comment, marked as answer]
https://docs.dagster.io/_apidocs/execution#dagster.reexecute_pipeline

The "parent run id" is the id of the previously failed run to execute. Our naming is not as good as it could be here.
